modules/qtwidgets/rawdatareaderdialogqt.cpp

```diff
@@ -43,27 +44,103 @@ namespace inviwo {
 RawDataReaderDialogQt::RawDataReaderDialogQt() {
     setWindowTitle("Importing Raw Data");
     QGridLayout* mainLayout = new QGridLayout(this);
+	mainLayout->setContentsMargins(15,15,15,15);
     QLabel* fileNameLabel = new QLabel("Importing file:");
-    fileName_= new QLabel();
+    fileName_ = new QLabel();
     QGridLayout* dataTypeLayout = new QGridLayout();
     QLabel* bitDepthLabel = new QLabel("Data format");
-    bitDepth_= new QComboBox();
+    bitDepth_ = new QComboBox();
     bitDepth_->addItem("UCHAR");
     bitDepth_->addItem("CHAR");
     bitDepth_->addItem("USHORT");
-    bitDepth_->addItem("USHORT_12");
+    bitDepth_->addItem("USHORT (12-bits)");
     bitDepth_->addItem("UINT");
     bitDepth_->addItem("INT");
     bitDepth_->addItem("FLOAT");
     bitDepth_->addItem("DOUBLE");
+	connect(bitDepth_, static_cast<void (QComboBox::*)(int)>(&QComboBox::currentIndexChanged), this, [=](int index) {
```

Comment:
Consider moving the code in the lambda to a member function since it makes this function quite long. Having it as a member function you can also call it instead of `bitDepth_->setCurrentIndex(2);` to set the default values.

Reply:
WIll do